even more prb testing

shaneknapp opened the pull request:
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)
(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

Please review http://spark.apache.org/contributing.html before opening a pull request.

shaneknapp committed:
even more prb testing

shaneknapp committed:
even more prb testing

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/78/
Test FAILed.

shaneknapp committed:
testing pyarrow for bryan cutler

shaneknapp committed:
testing pyarrow for bryan cutler

shaneknapp commented:
test this please

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/79/
Test FAILed.

shaneknapp committed:
removing extra whitespace from copypasta

shaneknapp committed:
removing extra whitespace from copypasta

srowen committed:
[SPARK-23654][BUILD] remove jets3t as a dependency of spark

## What changes were proposed in this pull request?

Remove jets3t dependency, and bouncy castle which it brings in; update licenses and deps
Note this just takes over https://github.com/apache/spark/pull/21146

## How was this patch tested?

Existing tests.

Closes #22081 from srowen/SPARK-23654.

Authored-by: Sean Owen <srowen@gmail.com>
Signed-off-by: Sean Owen <sean.owen@databricks.com>

zhengruifeng committed:
[SPARK-24555][ML] logNumExamples in KMeans/BiKM/GMM/AFT/NB

## What changes were proposed in this pull request?
logNumExamples in KMeans/BiKM/GMM/AFT/NB

## How was this patch tested?
existing tests

Closes #21561 from zhengruifeng/alg_logNumExamples.

Authored-by: zhengruifeng <ruifengz@foxmail.com>
Signed-off-by: Sean Owen <sean.owen@databricks.com>

shaneknapp committed:
i honestly cannot believe how long these builds take

shaneknapp committed:
i honestly cannot believe how long these builds take

dilipbiswal committed:
[SPARK-25092][SQL][FOLLOWUP] Add RewriteCorrelatedScalarSubquery in list of nonExcludableRules

## What changes were proposed in this pull request?
Add RewriteCorrelatedScalarSubquery in the list of nonExcludableRules since its used to transform correlated scalar subqueries to joins.

## How was this patch tested?
Added test in OptimizerRuleExclusionSuite

Author: Dilip Biswal <dbiswal@us.ibm.com>

Closes #22108 from dilipbiswal/scalar_exclusion.

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/80/
Test FAILed.

shaneknapp committed:
Merge branch 'more-ubuntu-testing' of github.com:shaneknapp/spark into more-ubuntu-testing

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/81/
Test FAILed.

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/ubuntuSparkPRB/82/
Test FAILed.
